setup.py

```diff
@@ -10,7 +10,9 @@
 
 # Get the version string
 with open(path.join(here, 'tianshou', '__init__.py')) as f:
```

Comment:
It is not the recommended way to handle the version number. You should rather `import tianshou` in `setup.py` itself, and assign `version` to `tianshou.__version__`.

Reply:
Nice catch!

Reply:
Oops, it cannot work: https://github.com/thu-ml/tianshou/pull/189/checks?check_run_id=1031970772

Reply:
Should I modify `tianshou/__init__.py` to only a single line:
```python
__version__ = '0.2.6'
```
?

Reply:
Ok I see the issue. Have a look to the good practice to doing what you are looking for on the official python documentation. There are several approches  including uding a "VERSION" file. It would be more standard than a ".txt" file even though it is basically the dame thing.